pkg/volume/csi/csi_attacher.go

```diff
@@ -255,7 +255,7 @@ func (c *csiAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.No
 
 		skip, err := c.plugin.skipAttach(driverName)
 		if err != nil {
-			klog.Error(log("Failed to check CSIDriver for %s: %s", driverName, err))
```

Comment:
What is purpose of this PR? There is IMO no difference between printing `error` as `%v` and `%s`.

Reply:
They have same results, just keep same log format style consistent, rather other than sometimes using `%s`, sometimes using `%v` for the same variable

Reply:
I think it's inevitable in a big project like this. AFAIK we don't have strict guidelines how `error` should be printed and different people prefer different style.

Reply:
Yep， you are right. But I think it’s better to keep consistent in the same func.